starknet-foundry/crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -476,6 +471,17 @@ fn declare(
     Ok(())
 }
 
+fn get_class_hash(casm_contract: &str) -> ClassHash {
+    let compiled_class = serde_json::from_str::<CompiledClass>(casm_contract)
+        .expect("Failed to deserialize casm contract into CompiledClass");
+    let class_hash = compiled_class
+        .class_hash()
+        .expect("Failed to compute class_hash for CompiledClass");
+    let class_hash = StarkFelt::new(class_hash.to_bytes_be())
+        .unwrap_or_else(|_| panic!("Failed to create StarkFelt from class_hash: {class_hash:?}"));
+    ClassHash(class_hash)
+}
```

Comment:
I think this should return `Result<ClassHash>` so we can handle it like other errors. Not by unwrapping.

Reply:
Errors should be bubbled `?`